infra/base-images/base-runner/run_fuzzer

```diff
@@ -118,7 +123,11 @@ elif [[ "$FUZZING_ENGINE" = honggfuzz ]]; then
   # -n: number of fuzzing threads (and processes)
   CMD_LINE="$OUT/honggfuzz -n 1 --exit_upon_crash -R /tmp/${FUZZER}_honggfuzz.report -W $FUZZER_OUT -v -z -P -f \"$CORPUS_DIR\" $(get_dictionary) $* -- \"$OUT/$FUZZER\""
 else
-  CMD_LINE="$OUT/$FUZZER $FUZZER_ARGS $* $CORPUS_DIR"
+  CMD_LINE="$OUT/$FUZZER $FUZZER_ARGS $*"
+
+  if [ -z ${DO_NOT_USE_CORPUS:-} ]; then
```

Comment:
`SKIP_SEED_CORPUS` indicates whether we unpack seed corpora or not. Even if we don't unpack it, we still pass an empty directory to the fuzzer, and that's the thing we need to avoid when reproducing a single testcase.

Reply:
Hm, although the corpus dir isn't really used anyhow afterwards, in such case we can rely on just one flag, let me try